HomeTask8/src/main/java/org/example/entity/GroupHW8.java

```diff
@@ -0,0 +1,51 @@
+package org.example.entity;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "group_hw8")
+public class GroupHW8 {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    @Column(name = "title")
+    private String title;
+    @OneToMany(mappedBy = "groupId")
```

Comment:
Does it correct here?
As I could see, you changed the name in StudentHW8, biy didn't chane it here